src/components/DicomControls.vue

```diff
@@ -4,34 +4,6 @@
       class="oc-background-brand oc-p-s oc-width-large oc-flex oc-flex-middle oc-flex-center oc-flex-around preview-controls-action-bar"
     >
       <!-- hide prev / next section while functionality is not yet implemented, set class of the wrapper div to 'oc-width-xlarge' once that section is activated -->
```

Comment:
the following code snipped was uncommented on purpose as the comment above states. If sonar cloud requires this to be removed, then also the comment above should be removed. Although I didn't notice that this caused an error in sonar cloud...

Reply:
the comment is removed